daemon/daemon_unix.go

```diff
@@ -550,7 +550,7 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 	// check for various conflicting options with user namespaces
 	if daemon.configStore.RemappedRoot != "" && hostConfig.UsernsMode.IsPrivate() {
 		if hostConfig.Privileged {
-			return warnings, fmt.Errorf("Privileged mode is incompatible with user namespaces")
+			return warnings, fmt.Errorf("Privileged mode is incompatible with user namespaces.  To temporarily disable user namespaces, use '--userns=host'.")
```

Comment:
@thaJeztah can we open a discussion on how to design that ? Possibly avoiding grepping for struct properties and bames. Maybedoibg like one could do with i18n could be a way but that would mean touching all the error messages coming from all over the repo.